src/status_im/extensions/ethereum.cljs

```diff
@@ -234,12 +234,31 @@
                         :on-failure on-failure}]
      (wrap-with-resolution db json-rpc-args :to execute-send-transaction))))
 
-(defn- parse-log [{:keys [address transactionHash blockHash transactionIndex topics blockNumber logIndex removed data]}]
-  (merge {:data    data
+(defn- event-topic-enc [event params]
+  (let [eventid (str event "(" (string/join "," params) ")")]
+    (abi-spec/sha3 eventid)))
+
+(defn diff-vectors [X Y]
+  (reduce #(remove (fn [x] (= x %2)) %1) X Y))
+
+(defn- decode-data [data hints firstTopic indexed]
```

Comment:
Usually dash based is preferred for naming things.

Reply:
Fixed in the new commit, Thanks!